<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchPointFrameCorrectionControls</class>
 <widget class="QWidget" name="MatchPointFrameCorrectionControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>850</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Loaded algorithm:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_lbLoadedAlgorithmName">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="NoBrush">
          <color alpha="128">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="NoBrush">
          <color alpha="128">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>159</red>
           <green>158</green>
           <blue>158</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="NoBrush">
          <color alpha="128">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Image:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkSingleNodeSelectionWidget" name="imageNodeSelector" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_TargetMask">
     <property name="text">
      <string>Target mask:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkSingleNodeSelectionWidget" name="maskNodeSelector" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="m_tabs">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="m_tabSelection">
      <attribute name="title">
       <string>Selection</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_Heading1">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Load an algorithm</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_algo">
         <property name="text">
          <string>Algorithm browser selection:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="m_pbLoadSelected">
         <property name="text">
          <string>Load selected algorithm</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_algoInfo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Selected algorithm information:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QmitkAlgorithmProfileViewer" name="m_teAlgorithmDetails" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="m_lbSelectedAlgorithm">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="midLineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>label_Heading1</zorder>
      <zorder>m_teAlgorithmDetails</zorder>
      <zorder>label_algo</zorder>
      <zorder>label_algoInfo</zorder>
      <zorder>m_pbLoadSelected</zorder>
      <zorder>m_lbSelectedAlgorithm</zorder>
     </widget>
     <widget class="QWidget" name="m_tabExecution">
      <attribute name="title">
       <string>Execution</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_Heading3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Perform frame correction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Correction job name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_leRegJobName">
         <property name="toolTip">
          <string>Name of the resulting mapped image</string>
         </property>
         <property name="text">
          <string>Unnamed RegJob</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="m_pbStartReg">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Progress</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Frames:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="m_progBarFrame">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="m_lbProgBarLevel">
            <property name="text">
             <string>Resolution level:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="m_progBarLevel">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="format">
             <string>%v/%m</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="m_lbProgBarIteration">
            <property name="text">
             <string>Iterations:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="m_progBarIteration">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="format">
             <string>%p% (%v/%m)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_log">
            <property name="text">
             <string>Log:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="m_teLog">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="lineWrapMode">
             <enum>QTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="m_checkClearLog">
              <property name="text">
               <string>Clear log on algorithm start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_pbSaveLog">
              <property name="text">
               <string>Save log to file</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabSettings">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Configure mapping parameters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QmitkMapperSettingsWidget" name="m_mapperSettings" native="true"/>
       </item>
       <item>
        <widget class="QLabel" name="label_Heading2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Configure algorithm parameters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QmitkAlgorithmSettingsConfig" name="m_AlgoConfigurator" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabExclusion">
      <attribute name="title">
       <string>Frame selection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select frames/timepoints that should be corrected:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="m_listFrames"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="m_btnFrameSelAll">
           <property name="text">
            <string>Select all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnFrameDeSelAll">
           <property name="text">
            <string>Deselect all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnFrameInvert">
           <property name="text">
            <string>Invert</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkSingleNodeSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkSingleNodeSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkAlgorithmProfileViewer</class>
   <extends>QWidget</extends>
   <header>QmitkAlgorithmProfileViewer.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkAlgorithmSettingsConfig</class>
   <extends>QWidget</extends>
   <header>QmitkAlgorithmSettingsConfig.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkMapperSettingsWidget</class>
   <extends>QWidget</extends>
   <header>QmitkMapperSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
